src/main/java/de/t0biii/music/domain/Music.java

```diff
@@ -32,7 +37,7 @@ public static void stop(UUID uuid) {
   }
 
   private static void play(Player player, Main plugin) {
-    if (player.hasPermission("JoinMusic.use") || player.isOp()) {
+    if ((player.hasPermission("JoinMusic.use") || player.isOp())) {
```

Comment:
i dont see the reasone for this change?

Reply:
I don't really know when did that happen